ethapi/backend.go

```diff
@@ -76,7 +78,26 @@ type Backend interface {
 	GetEvent(ctx context.Context, shortEventID string) (*inter.Event, error)
 	GetEventHeader(ctx context.Context, shortEventID string) (*inter.EventHeaderData, error)
 	GetConsensusTime(ctx context.Context, shortEventID string) (inter.Timestamp, error)
-	GetHeads(ctx context.Context, epoch int) (hash.Events, error)
+	GetHeads(ctx context.Context, epoch rpc.BlockNumber) (hash.Events, error)
+
+	// Lachesis DAG API
+	CurrentEpoch(ctx context.Context) idx.Epoch
```

Comment:
add a comment to shortly describe to these API.

Reply:
Those comments are listed in sfc_api.go file and ethapi_backend.go. This file is an interface